modules/F50_base_aggregator.sh

```diff
@@ -129,7 +152,11 @@ F50_base_aggregator() {
   fi
 
   if [[ "$STRCPY_CNT" -gt 0 ]]; then
+
     print_output ""
```

Comment:
Here you can use `print_output "\\n-----------------------------------------------------------------\\n"`